common/api/ecschema-locaters.api.md

```diff
@@ -19,38 +20,53 @@ export class FileSchemaKey extends SchemaKey {
     schemaText?: string;
 }
 
-// @beta
+// @alpha
```

Comment:
Is there a reason why you switched from beta to alpha?

Reply:
In one of the commits, I had to make the change from beta to alpha because `rush extract-api` was not working without this modification. Nonetheless, I'll roll back this change to beta.

Reply:
Rolled back the testing from alpha to beta